arm-storage/2015-06-15/swagger/storage.json

```diff
@@ -1,17 +1,19 @@
 {
   "swagger": "2.0",
   "info": {
-    "title": "StorageManagementClient",
-    "description": "The Storage Management Client.",
-    "version": "2015-06-15"
+    "title": "StorageManagement",
+    "description": "The Azure Storage Management API.",
+    "version": "2015-06-15",
+    "x-ms-code-generation-settings": {
+      "name": "StorageManagementClient"
+    }
   },
   "host": "management.azure.com",
   "schemes": [
     "https"
   ],
   "consumes": [
-    "application/json",
-    "text/json"
```

Comment:
Its just for consistency with other api-version swaggers. e.g the latest version does not have this.  https://github.com/Azure/azure-rest-api-specs/blob/master/arm-storage/2016-12-01/swagger/storage.json#L16